### PR TITLE
EZP-26834 : Add README and LICENCE

### PR DESCRIPTION
**Pull request by nicolas-bastien**
Jira : https://jira.ez.no/browse/EZP-26834

## Description

Add readme and licence files.

## Test

Visually by reading

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# EzPlatform I18N
+
+## Introduction
+
+This is a meta repository which centralize EzPlatform translations to ease synchronization with Crowdin.
```
Comment (bdunogier) [inline README.md]: I'd say "that centralizes".

File: README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+**Important** : This repository is not suppose to be used on a production project, each translation will be
```
Comment (bdunogier) [inline README.md]: Typo: "supposed".

File: README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+If you want to help us on translating EzPlatform feel free to contribute on [EzPlatform project on Crowdin][crowdin-ezplatform].
+
+### In context translation
+
+You can enable [Crowdin in context translation][in-context] by setting a specific cookie on a EzPlatform application.
+
+One way to do this is to open the development console (right click, inspect) and run these lines:
+
+- enable: document.cookie='ez_in_context_translation=1;path=/;'; location.reload();
```
Comment (bdunogier) [inline README.md]: Format as code, maybe ?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+- disable: document.cookie='ez_in_context_translation=;expires=Mon, 05 Jul 2000 00:00:00 GMT;path=/;'; location.reload();
+
+## Repository split
+
+Each locale will be stored in it's own repository in [ezplatform-i18n organisation][ezplatform-i18n-org].
```
Comment (bdunogier) [inline README.md]: "its"

File: README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+This synchronisation is done using [dflydev/git-subsplit][dflydev] tool.
+
+    git subsplit init https://github.com/react-php/react
```
Comment (bdunogier) [inline README.md]: react-php ? copy-paste, or am I missing something ?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    git subplit update
+    git subsplit publish "
+        fr_FR:git@github.com:ezplatform-i18n/ezplatform-i18n-fr_FR.git
+        ach_UG:git@github.com:ezplatform-i18n/ezplatform-i18n-ach_UG.git
+    " --heads=master
```
Comment (bdunogier) [inline README.md:39]: What about multiple branches ?

**Review by bdunogier — CHANGES_REQUESTED** [review]
Just a few minor remarks, good job !

File: README.md (hunk 6 above)
Comment (nicolas-bastien) [reply README.md:39]: Here we can have a list list that 

    --heads="master 2.0"

I guess we can set that during the next release

**Comment by nicolas-bastien** [comment]
@bdunogier done

You were right about the react thing so I've just retested all from a new folder to validate the process

Should be good now

File: README.md (hunk 6 above)
Comment (bdunogier) [reply README.md:39]: Okay.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# EzPlatform I18N
```
Comment (bdunogier) [inline README.md]: We usually spell it "eZ Platform" (see http://ezplatform.com). Also, it is usually spelled i18n (lowercase).

File: README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+## Introduction
+
+This is a meta repository that centralizes EzPlatform translations to ease synchronization with Crowdin.
+
+**Important** : This repository is not supposed to be used on a production project, each translation will be 
+splitted in a dedicated package.
+
+## Crowdin
+
+If you want to help us on translating EzPlatform feel free to contribute on [EzPlatform project on Crowdin][crowdin-ezplatform].
```
Comment (bdunogier) [inline README.md]: Capitalization of "eZ Platform".

File: README.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+### In context translation
+
+You can enable [Crowdin in context translation][in-context] by setting a specific cookie on a EzPlatform application.
```
Comment (bdunogier) [inline README.md]: Capitalization.

**Review by bdunogier — APPROVED** [review]
I'd just like the capitalization details fixed, then you can merge.

**Comment by nicolas-bastien** [comment]
@bdunogier done and merged

**Pull request merged**